dependencies.cfg

```diff
@@ -1,6 +1 @@
-# Required dependencies of this project
-# Each line specifies one dependency: <name><separator><version>
-#   name: dependency name
-#   separator: colon + optional white space
-#   version: git branch or tag
 integrations.library: v0.6.0
```

Comment:
I've created a new release v0.7.0 which includes are changes in integrations.library.
Please update once more and don't delete the comments.